docs/howto/openstack/octavia/lbaas-tcp.md

```diff
@@ -0,0 +1,742 @@
+    | domain_name         | None                                 |
+    | tags                |                                      |
+    +---------------------+--------------------------------------+
+    ```
+
+    The name of the new health monitor is `mylb-pool-healthmon`,
+    and the TCP protocol will be used to check whether members
+    of pool `mylb-pool` are online or offline. To check the
+    provisioning status of the health monitor, type:
+    
+    ```bash
+    openstack loadbalancer healthmonitor show \
+        mylb-pool-healthmon -c provisioning_status
+    ```
+    
+    ```plain
+    +---------------------+--------+
+    | Field               | Value  |
+    +---------------------+--------+
+    | provisioning_status | ACTIVE |
+    +---------------------+--------+
+    ```
+    
+    Once the health monitor is provisioned, you may at any time
+    check the pool members status like so:
+    
+    ```bash
+    openstack loadbalancer member list mylb-pool
+    ```
+    
+    ```plain
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    | id        | name      | project_id | provisioning_status | address   | protocol_port | operating_status | weight |
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    | c4351167- | srv-      | dfc7004673 | ACTIVE              | 10.15.25. |         61234 | ONLINE           |      1 |
+    | fcfa-     | lbaas-1   | 96428bacba |                     | 105       |               |                  |        |
+    | 48fb-     |           | 4376e72cc3 |                     |           |               |                  |        |
+    | 82ac-     |           | e9         |                     |           |               |                  |        |
+    | 76ddfdc73 |           |            |                     |           |               |                  |        |
+    | ca2       |           |            |                     |           |               |                  |        |
+    | 04a32731- | srv-      | dfc7004673 | ACTIVE              | 10.15.25. |         61234 | ONLINE           |      1 |
+    | f3a6-     | lbaas-2   | 96428bacba |                     | 236       |               |                  |        |
+    | 41ca-     |           | 4376e72cc3 |                     |           |               |                  |        |
+    | a638-     |           | e9         |                     |           |               |                  |        |
+    | 6f8767437 |           |            |                     |           |               |                  |        |
+    | 50c       |           |            |                     |           |               |                  |        |
+    +-----------+-----------+------------+---------------------+-----------+---------------+------------------+--------+
+    ```
```

Comment:
There's one more note I would include here (because it is frequently a source of confusion), which is what `openstack loadbalancer member list` looks like after you kill `ncat` on one server, and what your `wget` command does under the same circumstances (namely, that it hits only the remaining healthy server, rather than alternately hitting a healthy and a broken one).

Reply:
Ah, yes, that note would be useful. Adding and updating in a bit...

Reply:
Ack, and in the process please also fix [the Markdown lint errors](https://github.com/citynetwork/docs/actions/runs/3986415488/jobs/6834966865#step:6:33) that the CI runs highlight.

Reply:
Yeah, although I run `tox -e markdownlint` before updating, sometimes I accidentally open the file from the *wrong* editor, errors creep in, and of course, I forget to re-run the linter :/